beanstalk_worker/worker_start_scan.py

```diff
@@ -110,7 +110,8 @@ def run(self):
 
         # TODO: Figure out why random tag (with date) is coming
         # image_under_test = ":".join(self.job_info.get("name").split(":")[:-1])
-        image_under_test = self.job_info.get("name")
+        image_under_test = self.job_info.get("name") + ":" \
+            + self.job_info.get("tag")
```

Comment:
@dharmit : the same code will be executed for usual scan per build and weekly scan as well, I'd suggest you test this PR along with triggering usual scans per build as well and verify.